resources/views/exception.blade.php

```diff
@@ -1,59 +1,58 @@
-    <div x-data="{ active: 0 }" class="space-y-2 ">
-        @foreach ($this->frames as $index => $frame)
-            <div x-data="{
-                id: {{ $index }},
-                get section() {
-                    return this.active === this.id
-                },
-                set section(value) {
-                    this.active = value ? this.id : null
-                }
-            }"
-                class="rounded-lg bg-gray-100 dark:text-gray-200 dark:bg-gray-800 dark:border dark:border-gray-600">
-                <div class="flex flex-wrap break-all">
-                    <div x-on:click="section = !section"
-                        class="w-full flex items-center justify-start font-semibold text-sm pr-2 rtl:pl-2 py-2">
-                        <span x-show="section" aria-hidden="true" class="ml-4">
-                            <x-heroicon-o-eye class="h-6 w-6 text-primary-700 dark:text-primary-500 rtl:rotate-0" />
-                        </span>
-                        <span x-show="!section" aria-hidden="true" class="ml-4">
-                            <x-heroicon-o-eye-slash
-                                class="h-6 w-6 text-primary-700 dark:text-primary-500 rtl:rotate-180 " />
-                        </span>
-                        <span class="px-4 py-3 font-mono text-left break-words">
-                            {{ str($frame->file())->replace(base_path() . '/', '') }}
-                            in {{ $frame->method() }}
-                            at line
-                            <span
-                                class="inline-flex items-center justify-center ml-auto rtl:ml-0 rtl:mr-auto min-h-4 px-2 py-0.5 text-xs font-medium tracking-tight rounded-xl whitespace-normal text-primary-600 bg-primary-500/10 dark:text-primary-500">{{ $frame->line() }}</span>
-                        </span>
+<div x-data="{ active: 0 }" class="space-y-2 ">
+    @foreach ($this->frames as $index => $frame)
+        <div x-data="{
+            id: {{ $index }},
+            get section() {
+                return this.active === this.id
+            },
+            set section(value) {
+                this.active = value ? this.id : null
+            }
+        }"
+            class="bg-gray-100 rounded-lg dark:text-gray-200 dark:bg-gray-800 dark:border dark:border-gray-600">
+            <div class="flex flex-wrap break-all">
+                <div x-on:click="section = !section"
+                    class="flex items-center justify-start w-full py-2 pr-2 text-sm font-semibold rtl:pl-2">
+                    <span x-show="section" aria-hidden="true" class="ml-4">
+                        @svg('heroicon-o-eye', 'h-6 w-6 text-primary-700 dark:text-primary-500 rtl:rotate-0')
```

Comment:
i think instead of `@svg()` it's better to use the `<x-filament::icon />` component instead. it will even allow devs to change the icon through its alias
```
<x-filament::icon
    alias="fe::sectionOpen"
    class="w-6 h-6 text-primary-700 dark:text-primary-500 rtl:rotate-0"
    icon="heroicon-o-eye"
/>
```